rpc/client/local/local.go

```diff
@@ -219,6 +221,15 @@ func (c *Local) ProveShares(
 	return core.ProveShares(c.ctx, int64(height), startShare, endShare)
 }
 
+func (c *Local) ProveSharesV2(
+	ctx context.Context,
+	height uint64,
```

Comment:
[question] Why don't we initially define the type of `height` as int64?

Reply:
it's a choice that has been done when we started building Blobstream two years ago, and we never got around to change it to `int64`.